src/Sarif/Visitors/SarifCurrentToVersionOneVisitor.cs

```diff
@@ -231,6 +264,28 @@ internal RegionVersionOne CreateRegion(Region v2Region)
             return region;
         }
 
+        internal ReplacementVersionOne CreateReplacement(Replacement v2Replacement)
+        {
+            ReplacementVersionOne replacement = null;
+
+            if (v2Replacement != null)
+            {
+                replacement = new ReplacementVersionOne();
+
+                if (v2Replacement.InsertedContent != null)
+                {
+                    replacement.InsertedBytes = v2Replacement.InsertedContent.Text != null ?
+                                                    SarifUtilities.GetUtf8Base64String(v2Replacement.InsertedContent.Text) :
```

Comment:
This is a problem because we don't know the encoding of the file being modified, so we don't know if it's ok to stick a bunch of UTF-8 bytes into it.

I know we've run into this before -- do you remember where, and what we did about it?

Reply:
I think it was in one of the schema updates, I'll snoop around.

Reply:
I agree that an exception is appropriate, and I agree with your deferring the reporting. But I don't want to lose track of it; could you file an issue?